### ISSUE TITLE
TX intent fails to display when pubkey is given and CreateAccountOperation present in tx

### ISSUE THREAD

**Issue by salessandri** [stellar-expert/albedo#27]
I ran into the following issue. I have a transaction that I want the user to sign, this transaction contains a `CreateAccountOperation`. If the `pubkey` is not present in the params given to the `tx` intent, things are displayed correctly. But I want to restrict the user to signing with a specific public key. When I set the `pubkey` param, Albedo starts loading and then shows a blank page.

I debugged it a bit and found that it is making a request to horizon for the account being created and of course, that returns a 404. This seems to be triggering the failure.

This is the info I'm using:

- xdr: `AAAAAgAAAADWMDFohuQiZVyhhMYzYfnvFvzIqQvGQssoXRaFNQeycgAAAMgABcfNAAAAAAAAAAAAAAAAAAAAAgAAAAEAAAAA1jAxaIbkImVcoYTGM2H57xb8yKkLxkLLKF0WhTUHsnIAAAAAAAAAABsmKSe64xLVdnz/zobxnqpDmsEak+GLtGZaK7AnY0psAAAAAAX14QAAAAABAAAAABsmKSe64xLVdnz/zobxnqpDmsEak+GLtGZaK7AnY0psAAAABQAAAAAAAAAAAAAAAAAAAAEAAAACAAAAAQAAAAMAAAABAAAAAwAAAAEAAAADAAAAAAAAAAEAAAAA1jAxaIbkImVcoYTGM2H57xb8yKkLxkLLKF0WhTUHsnIAAAABAAAAAAAAAAA=`
- pubkey: `GDLDAMLIQ3SCEZK4UGCMMM3B7HXRN7GIVEF4MQWLFBORNBJVA6ZHFIJH`
- network: `testnet`

**Comment by salessandri** [comment]
It might actually be that there is a second operation targeting the account being created in the first operation and the signature of that key is not present. We just want a partial signature, we will add the other signature once the signed transaction is sent back to us.

**Comment by tyvdh** [comment]
I'm having this issue as well with partially signed transactions. No create account ops, just some payment and trustline stuff.

![Screen Shot 2020-10-02 at 8 43 58 AM](https://user-images.githubusercontent.com/4383610/94924870-4e44db00-048c-11eb-9a1e-cca9bff81f19.png)

pubkey:
```
GDJ4Z462LHYFDBMYGFTI2GX7BFMMCG7WCTSNDBA4RQR6I7GLF2WNQIO3
```
xdr:
```
AAAAAgAAAADTzPPaWfBRhZgxZo0a/wlYwRv2FOTRhByMI+R8yy6s2AAAASwAD5VoAAAABAAAAAEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAMAAAABAAAAAMin4RRGdDMjnKeCauKRjwqYsKkkGtPcEOj10Sn+AYToAAAAAQAAAADTzPPaWfBRhZgxZo0a/wlYwRv2FOTRhByMI+R8yy6s2AAAAAAAAAABKgXyAAAAAAAAAAAGAAAAAlNRMDAwMgAAAAAAAAAAAABRi+a3Ix5QqRhcC/Bqa+8wBYkQVlREPUvx07dipfHABwAAAAAAAAABAAAAAQAAAABRi+a3Ix5QqRhcC/Bqa+8wBYkQVlREPUvx07dipfHABwAAAAEAAAAA08zz2lnwUYWYMWaNGv8JWMEb9hTk0YQcjCPkfMsurNgAAAACU1EwMDAyAAAAAAAAAAAAAFGL5rcjHlCpGFwL8Gpr7zAFiRBWVEQ9S/HTt2Kl8cAHAAAAAAAAAAEAAAAAAAAAAqXxwAcAAABAcH8Kb/KizKD2xPNF75fVdUKibGEKiY1Tq4NyQfjmtgpXOKWrPFxhGwdUm/U7Oosqs5QYaPeDMHKHtspEWXvwBP4BhOgAAABAjGVZnczQYJRCPOIGhoGXimylYVytcM46gP/gtd6wCB/1oRbVeeApx3nvsmYIe/DfzfKN++EUPN5Frco33N9ECA==
```

**Comment by orbitlens** [comment]
Fixed in v0.9.3
Thanks for the heads up!
